docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -171,11 +233,11 @@ console.log(tx.toPretty());
 
 :::info
 
-MINA amounts, in all o1js APIs and elsewhere in the protocol, are always denominated in nanoMINA = `10^(-9)` MINA. This is why we set `const MINA = 1e9`.
+MINA amounts, in all o1js APIs and elsewhere in the protocol, are always denominated in nanoMINA = `10^(-9)` MINA (which is why we set `const MINA = 1e9`).
 
 :::
 
-What's interesting is that the transaction now has 3 account updates:
+The transaction now has these account updates:
```

Comment:
I didn't see three updates, I saw two... so I changed the word here. Is two accurate?